.github/ISSUE_TEMPLATE/new-feature.yml

```
@@ -0,0 +1,26 @@
+name: New Content
+description: Add documentation for a new feature or apply content suggestions
```

```suggestion
description: Suggest new content or a report documentation needed for a new feature.
```

Reword in a way that makes sense when creating a documentation issue.